src/PiggyCustomEnchants/EventListener.php

```diff
@@ -670,7 +672,8 @@ public function checkToolEnchants(Player $player, Event $event)
                 $finaldrop = array();
                 $otherdrops = array();
                 foreach ($drops as $drop) {
-                    if (isset(self::SMELTED_ITEM[$drop->getId()])) {
+                	$smelted = self::SMELTED_ITEM[$drop->getId()];
```

Comment:
If we're defining this variable, why not use it on line 677.

Reply:
was in a hurry, did the same for $tier